book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -0,0 +1,65 @@
+
+## Open & Responsible ML Licenses
+
+The "open source" approach to collaborative software development has permeated and influenced AI development and licensing practices. It is a common practice of ML developers to use open source licenses to release their ML models. This is due to the fact that open source licenses have become a standard practice when it comes to the sharing of artefacts in the entire ICT space (e.g., software; datasets; models; apps). ML developers might colloquially refer to "open sourcing a model" when they make its weights available by attaching an official open source license, or any other open software or content license such as Creative Commons. 
+
+However, open source licenses do not take the technical nature and capabilities of the ML model as a different artefact to software/source code into account, and are therefore ill-adapted to enabling a more responsible use of ML models.
+
+In order to balance the principles from open source with a growing demand of responsible ML development, use, and access, a new branch of ML licenses called Responsible AI Licenses (RAIL) emerged in 2019 with the [RAIL Initiative](https://www.licenses.ai/). Research initiatives such as [BigScience](https://bigscience.huggingface.co/) and companies such as [Hugging Face](https://huggingface.co/blog/open_rail) have decided to join efforts and push towards this direction along with the RAIL Initiative.
+
+Responsible AI licenses target specific concerns stemming from specific uses of an AI artefact by enacting use-based restrictions to mitigate potential harms associated with the use of AI-related products and services or component parts such as data, model, code, or applications. The integration of use-based restrictions clauses into open AI licenses brings up the ability to better control the use of AI artifacts and the capacity of enforcement to the licensor of the ML model, standing up for a responsible use of the released AI artifact, in case a misuse of the model is identified.
```

Comment:
How do we feel about adding information on guidance and support for enforcement? e.g. the Free Software Foundation and Bits of Freedom offer fund for legal counsel if a license is violated?

Reply:
I agree with this idea @arronlacey! Enforcement is the every-present problem behind licensing schemes, and FSF / BoF have been really influential.

Reply:
@arronlacey - thank you for this info! I will add a sentence to this chapter and hope to contribute to a future enforcement chapter with more info :D